components/_patterns/00-base/_outer-layout.scss

```diff
@@ -24,15 +16,16 @@ footer {
 
 .layout-sidebar {
   @include xl {
-    @include grid-span(1,1);
+    @include grid-span(4,1);
   }
 }
 
 .layout-content {
   @include xl {
-    @include grid-span(3,2);
+    @include grid-span(8,4);
```

Comment:
This should be 8,5 since the sidebar is using cols 1-4. And the sidebar was previously 1/4 of the screen. Now you have it taking up 1/3 of the screen is that too much? If we want it to match the previous grids layout-sidebar should be 3,1 and layout-content should be 9,4.